src/NovelRT.Interop/Graphics/NrtRenderingService.cpp

```diff
@@ -0,0 +1,235 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include "../NrtInteropErrorHandlingInternal.h"
+#include "NovelRT.Interop/Windowing/NrtWindowingService.h"
+#include "NovelRT.Interop/Graphics/NrtBasicFillRect.h"
+#include "NovelRT.Interop/NrtInteropUtils.h"
+#include "NovelRT.h"
```

Comment:
The more I see these the more I think we need a style guide. Source files have always used system includes for NovelRT headers in NovelRT itself, but as im writing these im beginning to question it.